discounts_5cb8501f2c7d3a18ac7ac51f.md

```diff
@@ -0,0 +1,27 @@
+At Semaphore, we offer discounts for non-profit, student and educational organizations.
+
+## Non-profit discounts
+
+For non-profit organizations, we offer a 25% discount. This means that at the end of each monthly billing cycle your total spending will be discounted at 25% off rate.
+
+If you are eligible for the discount, all you need to do is email us at [customersuccess@semaphoreci.com](mailto:customersuccess@semaphoreci.com) with a proof of your organization's status, such as the IRS determination letter for US based companies, or its equivalent for other countries.
+
+## Student accounts
+
+Every student currently enrolled at an accredited high school, college, or university, as well as any learn-to-code program or bootcamp, is eligible for a student account.
+
+Aside from regular $20 free credits per month, students will receive additional $240 gift credits each year, for as long as your studies last.
+
+In order to request a student account, please send us an email to [customersuccess@semaphoreci.com](mailto:customersuccess@semaphoreci.com) from your school-issued email address. In case you don’t have one, please provide some document in order to verify your status of a student.
+
+We will evaluate your request, and get back to you with an answer as soon as possible.
+
+## Educator accounts
```

Comment:
Can we add the following sentence on providing discounts for supporting edu internal projects:

Researchers or collaborators working on a project within any institution whose purpose directly relates to learning, teaching, and/or training, although may not directly be involved in the teaching process, are eligible for the discount.

Reply:
Maybe it can be a separate paragraph/topic.

Reply:
Call it **Research discounts** ?